llvm/utils/repo/docker/README

```diff
@@ -0,0 +1,41 @@
+Ability to create a Docker image for 'pstore'
```

Comment:
We're creating an image for the compiler so this should say "Ability to create a Docker image for llvm-prepo). (Remember that pstore is just the database back-end: the compiler is llvm-prepo.)

Reply:
Changed to llvm-prepo. Thanks for the clarification.